src/zk/zk_client.h

```diff
@@ -52,7 +53,11 @@ class ZkClient {
     ~ZkClient();
 
     // init zookeeper connections
-    bool Init();
+    // log level: DISABLE_LOGGING=0, ZOO_LOG_LEVEL_ERROR=1, ZOO_LOG_LEVEL_WARN=2, ZOO_LOG_LEVEL_INFO=3,
```

Comment:
add enum

Reply:
It's zk enum, integer is ok, no need to add more

Reply:
L44 in src/sdk/db_sdk.h